### DISCUSSION TITLE
Why is Image wrapped into a anonymous div?

### DISCUSSION THREAD

**Discussion by cangSDARM** [vercel/next.js#28552] [Help] [answered]
![image](https://user-images.githubusercontent.com/20441896/131094384-68599791-b584-4e78-8174-5afaad54a47d.png)

The red one used `next/Image`, and it's wrapped into a wired wrapper, which I cannot provide an id, class, or else to select to it. (always be img props)

The yellow one is a simple img element, which I can easily select it using css.

Why is Image wrapped into an anonymous, wired div? It's hard to write a CSS-driven and responsive page.

**Comment by guitheengineer** [comment, marked as answer]
Now it's a span, but the purpose is the same: to avoid layout shifting

> Regardless of the layout mode used, the Image component will have a consistent DOM structure of one `<img>` tag wrapped by exactly one `<span>`. For some modes, it may also have a sibling `<span>` for spacing. These additional `<span>` elements are critical to allow the component to prevent layout shifts.

https://nextjs.org/docs/basic-features/image-optimization

If you're having problems with styling this component, this docs section will help:
https://nextjs.org/docs/basic-features/image-optimization#styling
